backend/controllers/program.js

```diff
@@ -148,3 +148,20 @@ module.exports.deleteOne = (req, res) => {
 			});
 		});
 };
+
+module.exports.getFocusItemByProgram = (req, res) => {
+	FocusItem.find({program: req.params.id})
```

Comment:
We need to do an explicit query on the Program collection. The catch below will only execute on a mongoose failure. It won't catch the "missing program" error